aptos-move/aptos-e2e-comparison-testing/src/main.rs

```diff
@@ -3,14 +3,15 @@
 
 use anyhow::Result;
 use aptos_comparison_testing::{
-    prepare_aptos_packages, DataCollection, Execution, ExecutionMode, APTOS_COMMONS,
+    prepare_aptos_packages, DataCollection, Execution, ExecutionMode, OnlineExecutor, APTOS_COMMONS,
 };
 use aptos_rest_client::Client;
 use clap::{Parser, Subcommand};
+use move_core_types::account_address::AccountAddress;
 use std::path::PathBuf;
 use url::Url;
 
-const BATCH_SIZE: u64 = 100;
+const BATCH_SIZE: u64 = 500;
```

Comment:
Should this be configurable via cli? with a default value

Reply:
Will do in a separate PR.